### PR TITLE
CDAP-6069 Clarify instructions on how to run upgrade tool

### PR DESCRIPTION
**Pull request by awholegunch**
Adds a `sudo -u <cdap-user>` as appropriate.

Passes [Quick Build 2](http://builds.cask.co/browse/CDAP-DQB22-2)

Revised [Upgrade Instructions](http://builds.cask.co/artifact/CDAP-DQB22/shared/build-2/Docs-HTML/3.5.0-SNAPSHOT/en/admin-manual/upgrading/packages.html#admin-upgrading-packages)

Fix for https://issues.cask.co/browse/CDAP-6069

### CODE REVIEW

File: cdap-docs/admin-manual/source/upgrading/packages.rst [hunk 1]
```diff
@@ -27,9 +27,9 @@ upgrade instructions for the earlier versions and upgrade first to
 
 1. Stop all flows, services, and other programs in all your applications.
 
-#. Stop all CDAP processes::
+#. Stop all CDAP processes (as the user that runs CDAP Master, the CDAP user, indicated by ``<cdap-user>``)::
 
-     $ for i in `ls /etc/init.d/ | grep cdap` ; do sudo service $i stop ; done
+     $ for i in `ls /etc/init.d/ | grep cdap` ; do sudo -u <cdap-user> service $i stop ; done
```
Comment (poornachandra) [inline cdap-docs/admin-manual/source/upgrading/packages.rst]: Start and stop has to run as `sudo`. The init script downgrades the privileges before starting the java service.

**Comment by poornachandra** [comment]
@awholegunch Just one comment about not changing the commands for start/stop. Only upgrade command needs to be changed to use `sudo -u <cdap-user>`

File: cdap-docs/admin-manual/source/upgrading/packages.rst (hunk 1 above)
Comment (awholegunch) [inline cdap-docs/admin-manual/source/upgrading/packages.rst]: perhaps what you want is then; rather than `sudo`, use `su`:

```
$ su <cdap-user> -c 'for i in `ls /etc/init.d/ | grep cdap` ; do sudo service $i stop ; done'
```

Comment (wolf31o2) [inline cdap-docs/admin-manual/source/upgrading/packages.rst]: No. Poorna is correct. Revert this line change.

File: cdap-docs/admin-manual/source/upgrading/packages.rst [hunk 2]
```diff
@@ -91,11 +91,11 @@ upgrade instructions for the earlier versions and upgrade first to
    You can run the tool in a non-interactive fashion by using the ``force`` flag, in which case
    it will run unattended and not prompt for continuing::
    
-     $ /opt/cdap/master/bin/svc-master run co.cask.cdap.data.tools.UpgradeTool upgrade force
+     $ sudo -u <cdap-user> /opt/cdap/master/bin/svc-master run co.cask.cdap.data.tools.UpgradeTool upgrade force
      
 #. Restart the CDAP processes::
 
-     $ for i in `ls /etc/init.d/ | grep cdap` ; do sudo service $i start ; done
+     $ for i in `ls /etc/init.d/ | grep cdap` ; do sudo -u <cdap-user> service $i start ; done
```
Comment (wolf31o2) [inline cdap-docs/admin-manual/source/upgrading/packages.rst]: Revert this change.

File: cdap-docs/admin-manual/source/upgrading/packages.rst [hunk 3]
```diff
@@ -153,9 +153,10 @@ get upgraded correctly and HBase regionservers may crash.**
 
 #. After upgrading CDAP, start CDAP and check that it is working correctly.
 
-#. Stop all CDAP applications and services::
+#. Stop all CDAP applications and services (as the user that runs CDAP Master, the CDAP user, 
+   indicated by ``<cdap-user>``)::
    
-    $ for i in `ls /etc/init.d/ | grep cdap` ; do sudo service $i stop ; done
+    $ for i in `ls /etc/init.d/ | grep cdap` ; do sudo -u <cdap-user> service $i stop ; done
```
Comment (wolf31o2) [inline cdap-docs/admin-manual/source/upgrading/packages.rst]: Revert this, too.

File: cdap-docs/admin-manual/source/upgrading/packages.rst [hunk 4]
```diff
@@ -164,15 +165,15 @@ get upgraded correctly and HBase regionservers may crash.**
...
+    $ sudo -u <cdap-user> /opt/cdap/master/bin/svc-master run co.cask.cdap.data.tools.UpgradeTool upgrade_hbase
 
 #. Enable all CDAP tables; from an HBase shell, run this command::
 
     > enable_all 'cdap.*'
     
 #. Restart CDAP::
 
-    $ for i in `ls /etc/init.d/ | grep cdap` ; do sudo service $i start ; done
+    $ for i in `ls /etc/init.d/ | grep cdap` ; do sudo -u <cdap-user> service $i start ; done
```
Comment (wolf31o2) [inline cdap-docs/admin-manual/source/upgrading/packages.rst]: Revert this one.

**Comment by wolf31o2** [comment]
👍 LGTM... does this equivalent documentation exist for Cloudera Manager and Ambari, too?

**Comment by awholegunch** [comment]
Upgrading Apache Ambari and MapR are re-directed to these instructions. Cloudera Manager is upgraded through the UI, and these instructions do not apply.
